data/macros.meson

```diff
@@ -45,3 +45,9 @@
         --num-processes %{_smp_build_ncpus} \
         --print-errorlogs \
         %{nil}}
+
+# Macros for declarative buildsystems of rpm >= 4.20
```

Comment:
Please add a comment like in https://src.fedoraproject.org/fork/churchyard/rpms/pyproject-rpm-macros/blob/23ab565a7d3dd07e66eea2a317c9cf1e747f00f8/f/macros.pyproject

I guess this wasn't tested at all: `()` is needed for each macro.

Also, add `%buildsystem_meson_generate_buildrequires() %nil`. This way we avoid a potential issue if a different file defines an overlapping set of macros. It's better to get one set or the other.

I tested this with an example package (passim) and it seems to work fine.

I think we'd want to add a test case though.

Reply:
Does it work with packages more complex than passim, which use more features of meson?

Reply:
> Please add a comment like in https://src.fedoraproject.org/fork/churchyard/rpms/pyproject-rpm-macros/blob/23ab565a7d3dd07e66eea2a317c9cf1e747f00f8/f/macros.pyproject

Unfortunately, I could not use git command line now due to some network issues here. Is it possibile do a squash merge for this PR?

Reply:
I can do it via the command line... too many bad experiences with github's web UI makes me deeply distrust that button.